app/controllers/practices_controller.rb

```diff
@@ -139,7 +139,7 @@ def update
 
   def search
     ahoy.track "Practice search", {search_term: request.params[:query]} if request.params[:query].present?
-    @practices = Practice.where(approved: true, published: true).order(name: :asc)
+    @practices = Practice.left_outer_joins(:categories).select("practices.*, categories.name as categories_name").where(practices:{ approved: true, published: true }).order(name: :asc).uniq
     @facilities_data = facilities_json
```

Comment:
is this the same in the home_controller? can we make a method in the app controller to use in both places? or even in the practice model?

Reply:
ahh great idea will do! :)